src/components/form/submissions.vue

```diff
@@ -14,26 +14,30 @@ except according to the terms contained in the LICENSE file.
     <loading :state="keys.initiallyLoading"/>
     <page-section v-show="keys.dataExists">
       <template #heading>
-        <span>{{ $t('resource.submissions') }}</span>
-        <enketo-fill v-if="rendersEnketoFill" :form-version="form">
-          <span class="icon-plus-circle"></span>{{ $t('action.createSubmission') }}
-        </enketo-fill>
-        <template v-if="deletedSubmissionCount.dataExists">
-          <button v-if="canDelete && (deletedSubmissionCount.value > 0 || deleted)" type="button"
-            class="btn toggle-deleted-submissions" :class="{ 'btn-danger': deleted, 'btn-link': !deleted }"
-            @click="toggleDeleted">
-            <span class="icon-trash"></span>{{ $tcn('action.toggleDeletedSubmissions', deletedSubmissionCount.value) }}
-            <span v-show="deleted" class="icon-close"></span>
-          </button>
+        <div class="form-submissions-heading-row">
+          <enketo-fill v-if="rendersEnketoFill" :form-version="form">
+            <span class="icon-plus-circle"></span>{{ $t('action.createSubmission') }}
+          </enketo-fill>
+          <template v-if="deletedSubmissionCount.dataExists">
+            <button v-if="canDelete && (deletedSubmissionCount.value > 0 || deleted)" type="button"
+              class="btn toggle-deleted-submissions" :class="{ 'btn-danger': deleted, 'btn-link': !deleted }"
+              @click="toggleDeleted">
+              <span class="icon-trash"></span>{{ $tcn('action.toggleDeletedSubmissions', deletedSubmissionCount.value) }}
+              <span v-show="deleted" class="icon-close"></span>
+            </button>
+          </template>
+          <p v-show="deleted" class="purge-description">{{ $t('purgeDescription') }}</p>
+          <odata-data-access :analyze-disabled="analyzeDisabled"
+            :analyze-disabled-message="analyzeDisabledMessage"
+            @analyze="analyzeModal.show()"/>
+          <submission-download-button :form-version="form"
```

Comment:
I'm wondering if we could leave the download button in `SubmissionList`, but make it teleport here. Then the button would still be able to communicate with the download modal easily. Also, we wouldn't need to copy the `downloadDisabled` message here.

Just an idea!

Reply:
Just FYI, I've been working to teleport modals to a `<div>` in the `App` component. I ran into some issues in testing, but I was able to work around them. I wanted to mention the issues though in case you encounter something similar here. I'm hoping that my solution for teleporting modals will help in similar cases where there are issues in testing. I just have to clean up that commit, then I'll push a PR for it.